ajax/libs/timelinejs/package.json

```diff
@@ -1,30 +1,30 @@
 {
   "name": "timelinejs",
   "filename": "js/timeline-min.js",
-  "version": "2.36.0",
+  "version": "3.4.7",
   "description": "TimelineJS is an open-source tool that enables you to build visually-rich interactive timelines and is available in 40 languages.",
-  "homepage": "http://timeline.knightlab.com/",
+  "homepage": "https://timeline.knightlab.com/",
   "keywords": [
     "timeline"
   ],
+  "license": "MPL-2.0",
   "autoupdate": {
     "source": "git",
-    "target": "git://github.com/NUKnightLab/TimelineJS.git",
+    "target": "git://github.com/NUKnightLab/TimelineJS3.git",
     "fileMap": [
       {
         "basePath": "build",
         "files": [
-          "css/**/*",
-          "js/**/*"
+          "compiled/css/timeline*",
```

Comment:
Should include all file under `css/` and `js/` cause files are used in `.css` and `.js` files